### ISSUE TITLE
Elbow pronation contracture

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7690]
**Preferred term label:**
Elbow pronation contracture

**Synonyms**

**Definition (free text, please give PubMed ID)**
An elbow contracture that limits the ability of the elbow joint to supinate (supination refers to a rotation of the elbow jointt that causes the palm and forearm to face upwards), meaning that the palm is fixed in a downward orientation.

**Parent term (use hpo.jax.org/app)**
Elbow contracture (new term)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding as
Pronation contracture of the forearm
A contracture that limits the ability of the forearm to stabilize the hand in supiation (supination refers to a rotation of the forearm that causes the palm and forearm to face upwards), meaning that the palm is fixed in a downward orientation. Restriction of supination/pronation can occur at the elbow (radial-humeral joint), at the wrist (radial-ulnar joint), and anywhere in between.
